### ISSUE TITLE
User Review Token dashboard

### ISSUE THREAD

**Issue by nahuus123** [keep-network/keep-core#2083]
Found 2 minor things so added them here as well.

**Rewards - Beacon rewards** page. The ETH rewards decimals are overlapped by the status indicator 'accumulating;. Refer to below snapshot.

![image](https://user-images.githubusercontent.com/71384922/93883693-30d07e00-fce2-11ea-8402-a20d9a253061.png)

I would also propose to move the 'legend' of color indicators for available and accumulating to the table below as it applies there. It's now just below the total rewards number, which is always the accumulated sum of available and accumulating.

![image](https://user-images.githubusercontent.com/71384922/93884021-a3415e00-fce2-11ea-9738-0c383bdb49ae.png)

**Comment by r-czajkowski** [comment]
Hi!
Thanks for posting an issue!

Regarding:
> Rewards - Beacon rewards page. The ETH rewards decimals are overlapped by the status indicator 'accumulating;. Refer to below snapshot.

We are working on it. Pls see #2050

About the second suggestion I'll ping @liz-shinn

**Comment by liz-shinn** [comment]
Thanks for the feedback. The labels listed underneath the rewards balance are meant to show that number is the total accumulating + available balances. Will think on a better way to visually represent that.

A few questions:
1. Did they look clickable? Did you try to click them? If you did click them, what would you have expected them to do? My hunch is that if they're placed in the table near the title styled as they are now, they'll look like filters that sort the table instead of labels.

**Comment by nahuus123** [comment]
Hi Liz,

I did not feel they were clickable, did not feel the urge to click;)  I think if the field would be renamed to 'Total Rewards' or 'Total Rewards Balance' this would be enough, no need to add these labels.

There is already a drop down just above the table to filter. Maybe replacing this with the 3 colored buttons Total, Available and Accumulating would indeed combine the 2 functionalities of clarifying the colors and giving the option to filter in one go.

**Comment by liz-shinn** [comment]
Good to know! They weren't intended to look clickable, so that's good. Yes, the filter on the table will filter out the rows. I was referring to the label within the row, if maybe that should be clickable. But it's sounding like it's non-obvious to click it anyway.

Perhaps instead of showing those labels underneath could make a distinction between Total Rewards and Available Rewards and list those two amounts near each other. Stuff to think on 🤔

**Comment by liz-shinn** [comment]
Perhaps like this:

![PXL_20200922_160912840.jpg](https://user-images.githubusercontent.com/57226633/93911002-21612d00-fd02-11ea-973c-06e81d09293b.jpg)

![PXL_20200922_160921171.jpg](https://user-images.githubusercontent.com/57226633/93911010-245c1d80-fd02-11ea-9f09-f7fe0eee03cf.jpg)

**Comment by nahuus123** [comment]
Yeah, that would help! Is available the best word, or should it say 'released', 'transferable' or something of that nature?

**Comment by lionakhnazarov** [comment]
Closing as stale. Time-boxed community usability survey/review for the legacy KEEP token dashboard; the survey window closed years ago and the dashboard has moved to its own repo.
